docs/source/derived.rst

```diff
@@ -0,0 +1,111 @@
+.. _derived-waveforms:
```

Comment:
Can you add the example for NBI power/energy derived waveforms instead of only abstract examples?